extensions/amp-story/validator-amp-story.protoascii

```diff
@@ -258,6 +258,38 @@ tags: {
   attrs: { name: "animate-in-delay" }
   attrs: { name: "animate-in-duration" }
   attrs: { name: "grid-area" }
+  attrs: {
+    name: "scale-end"
+    value_regex: "^[0-9]*$"
+    value_properties: {
```

Comment:
It's not possible to have multiple value settings for an AttrSpec. It's an implicit oneof. That is, it can't be both `value_regex` and `value_properties`. Please see https://github.com/ampproject/amphtml/blob/master/validator/validator.proto#L105